src/Access/ContextAccess.cpp

```diff
@@ -266,6 +268,11 @@ namespace
 
     template <typename... OtherArgs>
     std::string_view getDatabase(std::string_view arg1, const OtherArgs &...) { return arg1; }
+
+    std::string_view getTableEngine() { return {}; }
```

Comment:
Delete this? Looks unused and I do not see why we need it

Reply:
It is like the one just 2 lines before it, to pass the compile of template since there exists a case with no argument.

Reply:
@kssenii could you take a look again, looks like the test failure is irrevalent?